src/Components/Header.jsx

```diff
@@ -7,14 +7,15 @@ const header = () => {
         <div className="header">
             <h1>NC Games</h1>
             <nav>
-                <a href="/">Home</a>
-                <a href="/account">Account</a>
-                <a href="/reviews">Reviews</a>
-                <a href="/categories">Categories</a>
-                <Link to="/comment">Comment</Link>
+                <ul>
+                    <li><a href="/">Home</a></li>
```

Comment:
use React Router's <Link> instead of <a> ... otherwise your whole page will re-render, and there's no point using React at all ... may as well just be a static site